UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -14,9 +14,9 @@ namespace UnofficialSteamAuthenticator
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    public sealed partial class App : Application
```

Comment:
I know you're just removing the inheritance, but an application being an application seems sensible, especially as it templated it like that when I made the app in VS.